docs/csharp/language-reference/keywords/is.md

```diff
@@ -63,15 +63,15 @@ Starting with C# 7.0, the `is` and [switch](../../../csharp/language-reference/k
 
 ### <a name="type" /> Type pattern </a>
```

Comment:
nit: I just noticed this, not related to your changes though. The anchor link terminates in a single statement and then follows the heading with another termination. I think the `</a>` gets stripped out though